server/controllers/request.js

```diff
@@ -0,0 +1,108 @@
+const Request = require("../models/Request");
+const User = require("../models/User");
+const asyncHandler = require("express-async-handler");
+
+// @route POST /request/
+// @desc create a new request by owner
+// @access Private
+exports.createRequest = asyncHandler(async (req, res, next) => {
+  try {
+    const user = await User.findById(req.user.id);
+    if (!user) {
+      return res
+        .status(401)
+        .json({ message: 'Not authorized' });
+    }
```

Comment:
Just to raise awareness, I'd say it's safe to assume `auth` middleware will properly catch any invalid tokens so this might be redundant.

Reply:
done